src/form/input/_input.scss

```diff
@@ -37,6 +37,6 @@
   }
 }
 
-.input--has-error {
+.input-container--has-error {
   border: 1px solid var(--invalid-text-color);
 }
```

Comment:
Do we want to give the border to the container or the input ? I guess adding it to the container is fine but let's make sure it is consistent

Reply:
I don't think it will be a problem. It works fine.
![image](https://user-images.githubusercontent.com/44462260/123755803-f5905a80-d8c4-11eb-81fa-cbfdd89474f7.png)